src/OmniSharp.MSBuild/MSBuildProjectSystem.cs

```diff
@@ -211,6 +217,24 @@ private static CSharpCompilationOptions CreateCompilationOptions(ProjectFileInfo
                 result = result.WithAllowUnsafe(true);
             }
 
+            var specificDiagnosticOptions = new Dictionary<string, ReportDiagnostic>(projectFileInfo.SuppressedDiagnosticIds.Count);
+
+            // Always suppress CS1701 (Assuming assembly reference 'x' used by 'y' matches identity 'z'. you may need to supply runtime policy)
+            specificDiagnosticOptions.Add("CS1701", ReportDiagnostic.Suppress);
```

Comment:
.net cli project system disables also `CS1702` and `CS1705` - should these be aligned?